evaluation/README.md

```diff
@@ -1 +1,87 @@
-# Evaluate your app
+# Evaluate Your AI App
+
+## Overall
+
+Evaluation is a critical process to measure the performance, quality and effectiveness of AI apps. This guide will help you to evaluate your AI apps and view the results in AI Toolkit.
+
+## Run evaluation
+### Step 1: Set up the Python environment
+
+1. Ensure Python (with version equal to or greater than 3.8) is installed.
+1. Install required libraries with the following command under project root:
+   ```bash
+   pip install -r ai-resource/evaluation/requirements.txt
+   ```
+
+### Step 2: Import your AI app as the evaluation target
+
+Import your AI app in the script and then wrap it into evaluation target. The evaluation target is a function accepts question and return an object of answer.
```

Comment:
Import your AI app into the script and wrap it in an evaluation target. The evaluation target is a function that takes a question and responds with an answer.